components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml

```diff
@@ -105,57 +105,58 @@ info:
     url: http://www.apache.org/licenses/LICENSE-2.0.html
   version: v2
 servers:
-- url: https://apis.wso2.com/api/am/publisher/v2
+  - url: https://apis.wso2.com/api/am/publisher/v2
```

Comment:
I would suggest not to do non-related formatting changes to this with this PR to publisher-api.yaml. If we are doing this let's do it in a separate PR and do the same in the master branch too.

Reply:
As discussed offline, we will send a PR to the master branch addressing all the formatting changes.